docs/setup_help/m1_mac_setup.md

```diff
@@ -0,0 +1,47 @@
+#Notes from Wes Goar on setup of vrs-python on a m1 mac
+- @author Wes Goar
+- date: 20220328
+- macOS version: 12.3
+- python version: 3.9.11
+
+##Notes
+- I installed seqrepo before trying this setup.
+- I had already installed xcode and homebrew on my machine.
+
+###Steps
+1. brew install openssl
+2. add the following statement in your .zshrc: export PATH="/opt/homebrew/opt/openssl@1.1/bin:$PATH"
+3. add the following statements in your .zshenv:
+4. Run brew info openssl to ascertain the correct environment path to put into the export statements
+
+export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" \
+export CPPFLAGS="-I/opt/homebrew/opt/openssl@3/include" \
+export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
+
+5. source ~/.zshrc
+6. source ~/.zshenv
+7. brew install libpq
+8. add the following statement in your .zshrc: export PATH="/opt/homebrew/opt/libpq/bin:$PATH"
+9. add the following statements in your .zshenv:
+10. Run brew info libpq to ascertain the correct environment path to append to the appropriate flags:
+
+export LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib -L/opt/homebrew/opt/libpq/lib"
+export CPPFLAGS="-I/opt/homebrew/opt/openssl@3/include -I/opt/homebrew/opt/libpq/include"
+export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig:/opt/homebrew/opt/libpq/lib/pkgconfig"
+
+11. source ~/.zshenv
+12. brew install postgres
+13. add the following statement in your .zshrc: export PATH="/opt/homebrew/opt/postgresql@14/bin:$PATH"
+    ###Make sure that you update the @14 with your own version
```

Comment:
You can check me on this, but I think all installed homebrew binaries that are arm-native should be symlinked in `/opt/homebrew/bin/` upon installation -- if that's correct, then you should be able to just make sure that's at the head of $PATH and you don't have to worry about adding particular packages with particular version numbers

Reply:
I guess we didnt really think about this, but what if people are using MacPorts instead of Homebrew?

Reply:
@reece or others may disagree, but in my experience Homebrew is effectively the default OSX package manager, and I think it's fine to only support the defaults (and anyone deviating from it should know enough to get through without as much assistance)

Reply:
I have no opinion about Mac stuff, but I support having instructions that help people for common environments. (I use linux everywhere.)

Reply:
James, 
I just checked the homebrew path comment on a different machine and it seemed to work like a charm. Now, this isn't an M1 based machine but I don't see why that would make a difference personally.